src/reporters/console/util.js

```diff
@@ -12,7 +12,8 @@ const CLEAR_RIGHT_OF_CURSOR = 1;
 export function clearLine(stdout: Stdout) {
   if (!supportsColor) {
     if (stdout instanceof tty.WriteStream) {
-      stdout.write(`\r${' '.repeat(stdout.columns - 1)}\r`);
+      const columns = stdout.columns > 0 ? stdout.columns : 100;
```

Comment:
I think this `100` number should be a constant in `constants.js` file.

Also a comment above this line about why we have this check would be great.